devicetypes/smartthings/zwave-fan-controller.src/zwave-fan-controller.groovy

```diff
@@ -217,4 +222,42 @@ def refresh() {
 
 def ping() {
 	refresh()
+}
+
+def getValueFor3SpeedDevice(rawLevel) {
```

Comment:
One more recommendation... Let's use `getFanSpeedFor3SpeedDevice` and `getFanSpeedFor4SpeedDevice` so that it is more clear what `Value` is.